siatune/tune/tuner.py

```diff
@@ -34,11 +36,12 @@ class Tuner:
             Refer to :class:`ray.tune.callback.Callback` for more info.
         resume (str, optional): The experiment path to resume.
             Default to None.
+        cfg (dict, optional) Full config. Default to None.
     """
 
     def __init__(
         self,
-        trainable: Callable[[dict], Any],
+        task: dict,
```

Comment:
How about `task_cfg` as in the line 44?

Reply:
To my current thought, all of `Callable` can be given by defined `Task`s.
So I don't feel the necessity to support `Callable` in the current status.